third-party/sqlalchemy-enum-tables/.gitlab-ci.yml

```diff
@@ -0,0 +1,69 @@
+stages:
```

Comment:
will this still be triggered if it's named `gitlab` instead of `github`?

Reply:
No, it won't but there really aren't any tests worth running.